### DISCUSSION TITLE
FTP文件乱码问题

### DISCUSSION THREAD

**Discussion by huami1314** [AlistGo/alist#4698] [General]
请问怎么解决 如题
![C3D3D473-2A4C-4AD1-BF30-1116D757CE5D](https://github.com/alist-org/alist/assets/99124359/3fa8b942-3f5e-4dd5-b72a-df5cb0892b20)

**Comment by wangkun19930608** [comment]
你的应该根据参数这是一下编码就可以。Encoding 设置为utf-8或者gbk试试。

*我再补充一个ftp的时区问题吧。
![image](https://github.com/user-attachments/assets/d5d020f8-1a1d-4862-8e32-d479367411b9)

**Comment by huami1314** [comment]
**感谢解答，差点忘记了**
